### PR TITLE
Feature/SK-728 | Move Studio docs to Fedn docs

### PR DESCRIPTION
**Pull request by niklastheman**
# Move Studio docs to Fedn docs

- Added Studio docs
- Expanded the APIClient docs
- Uses custom css to style in Scaleout fashion
- Uses Manrope font
- Uses Scaleout favicon

If you have time to test this locally, please do... Use this sphinx command to build: sphinx-build . _build and then run a http server from the _build directory.

### CODE REVIEW

File: docs/studio.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+.. image:: img/studio_project_overview.png
+
+
+Configure and attach clients
+----------------------------
+
+This step is very similar to how connecting a client to a local FEDn instance (:doc:`quickstart`). The main difference is that the client needs an authorization token to connect to a federation hosted on FEDn Studio. These are the steps to configure and attach a client to a project:
+
+In order to generate some data for training, on your local machine, navigate to the mnist-pytorch example:
```
Comment (ahellander) [inline docs/studio.rst:42]: In order to set up the local environment and configure local client data access, ...

File: docs/studio.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+Run the following commands:
+
+.. code-block:: bash
+
+    bin/init_venv.sh
+    bin/get_data
+    bin/split_data
+    bin/build.sh
+
+This commands will generate some training data and two files: package.tgz and seed.npz. The first is a compute package file and the second is a model file that can be used as the inital seed model. (For a more detailed explanation of the commands, see the :doc:`quickstart`)
```
Comment (ahellander) [inline docs/studio.rst:61]: These commands will generate training data..

**Review by ahellander — CHANGES_REQUESTED** [review]

File: docs/studio.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
+Studio
+===============
+
+FEDn Studio is a web-based tool for managing and monitoring federated learning experiments. It provides a user-friendly interface for creating and managing federated learning experiments, monitoring the progress of training, and visualizing the results.
```
[the remark this answers is not in the document]
Comment (niklastheman) [reply docs/studio.rst]: Done

File: docs/studio.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+Getting started
+---------------
+
+Before you can start using Studio, you will need an account. Head over to `studio.scaleoutsystems.com/signup <https://studio.scaleoutsystems.com/signup/>`_  and sign up today!
+
+Create a project
+----------------
+
+Start by creating a new project. A project is a container for your federated learning experiments. You can create multiple projects to organize your experiments. A project can be shared with other users, allowing you to collaborate on experiments.
```
[the remark this answers is not in the document]
Comment (niklastheman) [reply docs/studio.rst]: Done

File: docs/studio.rst (hunk 1 above)
Comment (niklastheman) [reply docs/studio.rst]: Done

File: docs/studio.rst (hunk 2 above)
Comment (niklastheman) [reply docs/studio.rst]: Done

File: docs/apiclient.rst [hunk 5]
```diff
@@ -3,6 +3,8 @@ APIClient
 
 FEDn comes with an *APIClient* for interacting with the FEDn network. The APIClient is a Python3 library that can be used to interact with the FEDn network programmatically. 
 
+Connect to the network
+----------------------
 
 The APIClient is available as a Python package on PyPI, and can be installed using pip:
```
Comment (ahellander) [inline docs/apiclient.rst:9]: Confusing, the APIClient is part of the fedn package, which is installed from pip?

File: docs/apiclient.rst [hunk 6]
```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
    from fedn import APIClient
    client = APIClient("localhost", 8092)
 
+Inital setup
```
Comment (ahellander) [inline docs/apiclient.rst:23]: Set the active compute package and the active seed model

File: docs/apiclient.rst [hunk 7]
```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
...
+------------
+
+Before you can start training models, you need to set the active package and an initail seed model. The active package can be set using the following code snippet:
```
Comment (ahellander) [inline docs/apiclient.rst:26]: Suggest to delete the sentence "Before you can start..."

File: docs/apiclient.rst [hunk 8]
```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
...
+
+.. code-block:: python
+   
+   client.set_active_package(path="path/to/compute_package.zip", helper="numpyhelper")
```
Comment (ahellander) [inline docs/apiclient.rst:30]: I think packages need to be tar.gz, tgz ? Plz check.

File: docs/apiclient.rst [hunk 9]
```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
...
+
+To set the initial seed model, you can use the following code snippet:
+
+.. code-block:: python
+   
+   client.set_active_model(path="path/to/seed_model.zip")
```
Comment (ahellander) [inline docs/apiclient.rst:36]: They are not usually zipped. Usually ends with ".npz" (numpy compressed)

File: docs/apiclient.rst [hunk 10]
```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
...
+
+Start training
```
Comment (ahellander) [inline docs/apiclient.rst:38]: Start training session

File: docs/apiclient.rst [hunk 11]
```diff
@@ -18,4 +20,60 @@ To initialize the APIClient, you need to provide the hostname and port of the FE
...
+Once the active package and seed model are set, you can connect clients to the network and start training models. The following code snippet initializes a session (training rounds):
+
+.. code-block:: python
+   
+   session = client.start_session(id="session_name")
+
+List data
+---------
+
+Other than starting training sessions, the APIClient can be used to get data from the network, such as sessions, models etc. All entities are represented and the all work in a similar fashion.
```
Comment (ahellander) [inline docs/apiclient.rst:50]: the -> they work

File: docs/studio.rst [hunk 12]
```diff
@@ -0,0 +1,94 @@
+Studio
+===============
+
+FEDn Studio is a web-based tool for managing and monitoring federated learning experiments. It provides a user-friendly interface for monitoring the progress of training and visualizing the results.
```
Comment (ahellander) [inline docs/studio.rst:4]: Maybe add / harmonize Studio intro with features mentioned in the new main README.

File: docs/studio.rst [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+
+Getting started
+---------------
+
+Before you can start using Studio, you will need an account. Head over to `studio.scaleoutsystems.com/signup <https://studio.scaleoutsystems.com/signup/>`_  and sign up today!
```
Comment (ahellander) [inline docs/studio.rst:9]: "..sign up today!" -> "sign up."

File: docs/studio.rst [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+2. Continue by clicking the "Create button". The FEDn template contains all the services necessary to start a federation.
+3. Enter the project name (mandatory). The project description is optional.
+4. Click the "Create" button to create the project.
+
+Project overview
+----------------
+
+Once you have created a project, you can find it via the sidebar link Projects. Here you will find the list of all your projects. When inside a project you can see the following tabs in the sidebar:
+
+1. **Dashboard**: The dashboard provides an overview of the project. The controller and combiner(s) are listed under "Network". This is also where you can find the current FEDn version and get the option to upgrade to a newer version if available
```
Comment (ahellander) [inline docs/studio.rst:26]: Add punctuation to last sentence in all items.

File: docs/studio.rst [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+3. **Combiners**: a list of combiners. Observe number of active clients for each combiner
+4. **Sessions**: a list of sessions with related models. Configure and start a new session. Upload compute package and seed model, set number of rounds, timeout limit etc
+5. **Models**: a list of models generated across sessions
+6. **Events**: a log of events from the combiner and the clients of the federated network
+7. **Settings**: project settings, including the option to delete the project and give access to other users
+
+.. image:: img/studio_project_overview.png
+
+
+Configure and attach clients
```
Comment (ahellander) [inline docs/studio.rst:37]: This covers two things, "Initialize the project with a compute package and a seed model", and "connect a client".

File: docs/studio.rst [hunk 16]
```diff
@@ -0,0 +1,94 @@
...
+In the Studio UI, navigate to the project you created and click on the "Sessions" tab. Click on the "New Session" button. Under the Compute package tab, select a name and upload the generated package file. Under the Seed model tab upload the generated seed file:
+
+.. image:: img/upload_package.png
+
+Connect client
+~~~~~~~~~~~~~~
+
+Navigate to clients and click on the "Download config" button. This downloads a client config file. Place this file in the mnist-pytorch directory on your local machine. Rename the file to client.yaml. Then edit it to set the name of the client.
+
+Later, in Studio, click on the "Connect client" button.
```
Comment (ahellander) [inline docs/studio.rst:75]: I think we should update this with "native instructions" also.

**Review by ahellander — CHANGES_REQUESTED** [review]

File: docs/studio.rst (hunk 16 above)
Comment (niklastheman) [reply docs/studio.rst:75]: Would like to leave this as is for now... Just until we've settled on a client connect view in studio.

File: docs/apiclient.rst (hunk 5 above)
Comment (niklastheman) [reply docs/apiclient.rst:9]: changed

File: docs/apiclient.rst (hunk 6 above)
Comment (niklastheman) [reply docs/apiclient.rst]: changed

File: docs/apiclient.rst (hunk 8 above)
Comment (niklastheman) [reply docs/apiclient.rst]: changed

File: docs/apiclient.rst (hunk 9 above)
Comment (niklastheman) [reply docs/apiclient.rst]: changed

File: docs/studio.rst (hunk 12 above)
Comment (niklastheman) [reply docs/studio.rst]: changed

File: docs/apiclient.rst (hunk 10 above)
Comment (niklastheman) [reply docs/apiclient.rst:40]: changed

File: docs/apiclient.rst (hunk 11 above)
Comment (niklastheman) [reply docs/apiclient.rst]: done

File: docs/studio.rst (hunk 13 above)
Comment (niklastheman) [reply docs/studio.rst]: changed

File: docs/studio.rst (hunk 14 above)
Comment (niklastheman) [reply docs/studio.rst]: done

File: docs/studio.rst (hunk 15 above)
Comment (niklastheman) [reply docs/studio.rst]: changed

**Review by ahellander — APPROVED** [review]

**Pull request merged**